src/SumatraUnitTests.cpp

```diff
@@ -1,20 +1,36 @@
 /* Copyright 2021 the SumatraPDF project authors (see AUTHORS file).
    License: GPLv3 */
 
+extern "C" {

```

Comment:
I forgot to remove those useless imports, this is why the build broke.
I'm on it.